### ISSUE TITLE
No dictionary for "German" availible on the new AMO page

### ISSUE THREAD

**Issue by Etamuk** [mozilla/addons#528]
I did a clean install of firefox 56.0.2. So i wanted to install the german dictionary. I went to "Addons" and got the new Addons-Page. But on this new site i couldn't find the german dictionary. On the old site it is still available. Please see my screenshots. I hope the german dictionaries will be copied to the new site.

**New site:**
![bildschirmfoto vom 2017-11-05 19-11-47](https://user-images.githubusercontent.com/20098065/32426213-0f3e643e-c2b9-11e7-8145-90f03efc6da2.png)

**Old site:**
![bildschirmfoto vom 2017-11-05 19-12-54](https://user-images.githubusercontent.com/20098065/32426216-161a04d4-c2b9-11e7-9a1c-ee87c0e58301.png)

**Comment by tofumatt** [comment]
They are still available but it seems out new site's mapping is missing some dictionaries. I'll have a look into why this is. Thanks for filing!

**Comment by muffinresearch** [comment]
@tofumatt when you have a moment please could you investigate this and see why the data is different?

**Comment by tofumatt** [comment]
The data returned is correct (I think) but we use the languages in Firefox's own `languages.json` which should have all known langs to us in it. This lets us organise the dictionaries like the old site does... it seems something isn't mapping and being omitted.

**Comment by muffinresearch** [comment]
@tofumatt is this something that could be in next week's milestone?

**Comment by tofumatt** [comment]
As in landed for today? I can certainly give it a go!

- Matt (Sent from mobile)

**Comment by muffinresearch** [comment]
> As in landed for today? I can certainly give it a go!

"next week" as-in next week's push rather than this week's push. :)

**Comment by RickieES** [comment]
Just for the record, is proper bug nettiquete to close always the latter bugs filed, not the first one.

**Comment by tofumatt** [comment]
Usually it’s best to just close the bug with the least info 😅

**Comment by RickieES** [comment]
Both bugs have the same amount of info, if you look at them. OP in this issue mentions German as missing, issue #523 mentions all es-* variants missing and some other languages missing at the top of the table. Both bugs mention where the problem may be (the JSON used to build the list).

The only extra information in this issue is that the AMO team does the triage late in the bug fix, the first you must do with a bug is search if it is a duplicate.

I just want the bug fixed, I'm not looking for recognition through my bug prevailing (I don't need or want that recognition after +15 years contributing Mozilla). I'm just pointing that bug triaging and marking for duplicates was the first task within a bug in Mozilla for a reason, which is avoiding duplicate issues going forward in parallel by different people in the team or outside of it.

**Comment by tofumatt** [comment]
I just checked the response data and it looks like the `target_locale` for a lot of the german-language dictionaries, etc, are things like `"target_locale": "de-DE"` not `"target_locale": "de"`. Might be this is messing up our filtering. I'm logging this here as I'm going away on holiday tomorrow and will only be working on this for a bit of time on the train. 😄

**Comment by tofumatt** [comment]
Figured out a fix for this and have it working, just need to write a few tests and then we're golden.

**Comment by Speravir** [comment]
I noticed this issue for German, too, but didn't know first, where to leave a message. But I also noticed that other languages are also affected, not just Spanish like in the closed ticket. Because according to rumours the now legacy design is going not to be available in some days I did a comparison _(something that should in my opinion have be done by someone of the developers before setting the new design as default …)_. Not present in new design, many of them may be automatically displayed again after the mentioned patch is applied, but for some it is not obvious for me, why they are hidden (note that I looked with Fx 56.x):
* [Acholi (UG) Language Pack](https://addons.mozilla.org/firefox/addon/acholi-ug-language-pack/)
* [Akan Spelling Dictionary](https://addons.mozilla.org/firefox/addon/akan-ns%C9%9Bmfuaasekyer%C9%9B/)
* [Amharic Spell Checker](https://addons.mozilla.org/firefox/addon/amharic-spell-checker/)
* [Corrector ortografico aragonés ](https://addons.mozilla.org/firefox/addon/corrector-ortografico-aragones/) (Aragonese dictionary)
* [Armenian (hy-AM) Language Pack](https://addons.mozilla.org/firefox/addon/armenian-hy-am-language-pack/) and  
  [Armenian spell checker dictionary](https://addons.mozilla.org/firefox/addon/armenian-spell-checker-diction/)
* [Assamese (IN) Language Pack ](https://addons.mozilla.org/firefox/addon/assamese-in-language-pack/) and  
  [Lachit - an Assamese Spell Checker](https://addons.mozilla.org/firefox/addon/assamese-spell-checker/)
* [Azerbaijani (AZ) Language Pack](https://addons.mozilla.org/firefox/addon/azerbaijani-az-language-pack/) and  
  [Azerbaijani Spell Checker](https://addons.mozilla.org/firefox/addon/azerbaijani-spell-checker/)
* [Беларуская Language Pack](https://addons.mozilla.org/firefox/addon/%D0%B1%D0%B5%D0%BB%D0%B0%D1%80%D1%83%D1%81%D0%BA%D0%B0%D1%8F-language-pack/),  
  [Belarusian Dictionary](https://addons.mozilla.org/firefox/addon/belarusian-dictionary/) and  
  [Belarusian Classic Dictionary](https://addons.mozilla.org/firefox/addon/belarusian-classic-dictionary/)
* [Bengali (India) Language Pack](https://addons.mozilla.org/firefox/addon/bengali-india-language-pack/)
* [Bosanski Language Pack](https://addons.mozilla.org/firefox/addon/bosanski-language-pack/) (Bosnian)
* [Breton Language Pack for Firefox](https://addons.mozilla.org/firefox/addon/breton-language-pack-for-firef/) and  
  [Difazier An Drouizig](https://addons.mozilla.org/firefox/addon/difazier-an-drouizig/) (Breton/Brezonheg dictionary)
* [Firefox en valencià](https://addons.mozilla.org/firefox/addon/valencian-catalan-languag-9702/) (Valencian Catalan language pack, AVL version),  
  [Diccionario ortográfico valenciano](https://addons.mozilla.org/firefox/addon/valencian-spellchecker/) (myspell version) and  
  [Diccionari valencià](https://addons.mozilla.org/firefox/addon/valencian-catalan-dictionary/) (AVL version)
* [Chichewa Spell Check](https://addons.mozilla.org/firefox/addon/chichewa-spell-check/)
* [Hrvatski (HR) Language Pack](https://addons.mozilla.org/firefox/addon/hrvatski-hr-language-pack/) (Croatian),  
  [Croatian Dictionary - Hrvatski Rjecnik](https://addons.mozilla.org/firefox/addon/croatian-dictionary-hrvatski-r/) and  
  [Croatian Dictionary (Hrvatski Rjecnik)](https://addons.mozilla.org/firefox/addon/croatian-dictionary-hrvat-9795/) (hr-HR)
* [Diccionario Papiamento](https://addons.mozilla.org/firefox/addon/diccionario-papiamento/) (pap-AW) and  
  [Dikshonario Papiamentu](https://addons.mozilla.org/firefox/addon/dikshonario-papiamentu/) (pap-CW)
* [English (Australian) Dictionary](https://addons.mozilla.org/firefox/addon/english-australian-dictionary/)
* [Canadian English Dictionary](https://addons.mozilla.org/firefox/addon/canadian-english-dictionary/)
* [New Zealand English Dictionary](https://addons.mozilla.org/firefox/addon/new-zealand-english-dict/)
* [English (South African) Language Pack](https://addons.mozilla.org/firefox/addon/english-south-african-language/)
* [Esperanto Language Pack](https://addons.mozilla.org/firefox/addon/esperanto-language-pack/) and  
  [Esperanta Vortaro](https://addons.mozilla.org/firefox/addon/esperanta-vortaro/)
* [Eesti (et) keele pakk](https://addons.mozilla.org/firefox/addon/eesti-et-keele-pakk/) (Estonian language pack) and  
  [Eesti keele speller](https://addons.mozilla.org/firefox/addon/eesti-keele-speller/) (Estonian dictionary)
* [Eʋegbe Spelling Dictionary](https://addons.mozilla.org/firefox/addon/e%CA%8Begbe-spelling-dictionary/)
* [Faroese Spell Checker (Faroe Islands)](https://addons.mozilla.org/firefox/addon/faroese-spell-checker-faroe-is/)
* [Dizionari Furlan](https://addons.mozilla.org/firefox/addon/dizionari-furlan/) (Friulian)
* [Pulaar Fulfulde Language Pack](https://addons.mozilla.org/firefox/addon/pulaar-fulfulde-language-pa/) (Fulah)
* [Corrector de Galego](https://addons.mozilla.org/firefox/addon/corrector-de-galego/) (Galician)
* German (Austria): [German Dictionary, extended for Austria](https://addons.mozilla.org/firefox/addon/german-dictionary-extended-for/) (by KaiRo) and  
  [German dictionary (de_AT)](https://addons.mozilla.org/firefox/addon/german-dictionary-de_at-for-sp/) (by B. Jacke)
* German (Germany): [German Dictionary](https://addons.mozilla.org/firefox/addon/german-dictionary/) (KaiRO, new spelling),  
  [German dictionary (de_DE)](https://addons.mozilla.org/firefox/addon/german-dictionary-de_de-for-sp/) (B. Jacke, new spelling) and  
  [German Dictionary, classical spelling standards](https://addons.mozilla.org/firefox/addon/deutsch-alte-rechtschreibung/) (T. Kuban)
* German (Swiss): [German Dictionary (Switzerland)](https://addons.mozilla.org/firefox/addon/german-dictionary-switzerland/) (KaiRo) and  
  [German dictionary (de_CH)](https://addons.mozilla.org/en-US/firefox/addon/german-dictionary-de_ch-for-sp/) (B. Jacke)
* [English - Greek Spelling dictionary](https://addons.mozilla.org/firefox/addon/english-greek-spelling-diction/)
* [Gujarati (IN) Language Pack](https://addons.mozilla.org/firefox/addon/gujarati-in-language-pack/) and  
 [Gujarati Spell Checker (India)](https://addons.mozilla.org/firefox/addon/gujarati-spell-checker-india/)
* [Haitian Creole spell checker](https://addons.mozilla.org/firefox/addon/haitian-creole-spell-checker/)
* [Hausa Spelling Dictionary](https://addons.mozilla.org/firefox/addon/hausa-spelling-dictionary/)
* [Hawaiian spell checker](https://addons.mozilla.org/firefox/addon/hawaiian-spell-checker/)
* [Hiligaynon spell checker](https://addons.mozilla.org/firefox/addon/hiligaynon-spell-checker/)
* [Hindi Language Pack](https://addons.mozilla.org/firefox/addon/hindi-language-pack/) and  
  [Hindi Spell Checker](https://addons.mozilla.org/firefox/addon/hindi-spell-checker/)
* [Icelandic (IS) Language Pack](https://addons.mozilla.org/firefox/addon/icelandic-is-language-pack/) and  
  [Icelandic Dictionary](https://addons.mozilla.org/firefox/addon/icelandic-dictionary/)
* [Interlingue Spell Checker](https://addons.mozilla.org/firefox/addon/interlingue-spell-checker/)
* [Taqbaylit Language Pack](https://addons.mozilla.org/firefox/addon/taqbaylit-language-pack/) (Kabyle)
* [Kannada (IN) Language Pack](https://addons.mozilla.org/firefox/addon/kannada-in-language-pack/) and  
  [Kannada Spell Checker – Add-ons for Firefox](https://addons.mozilla.org/firefox/addon/kannada-spell-checker/)
* [Kashubian Spell Checker (Poland)](https://addons.mozilla.org/firefox/addon/kashubian-spell-checker-poland/)
* [Kazakh (KZ) Language Pack](https://addons.mozilla.org/firefox/addon/kazakh-kz-language-pack/)
* [ខ្មែរ Language Pack](https://addons.mozilla.org/firefox/addon/%E1%9E%81%E1%9E%98%E1%9E%9A-language-pack/) (Khmer)
* [Rastnivîs](https://addons.mozilla.org/firefox/addon/rastniv%C3%AEs/) (Kurdish)
* [Lao Spellchecking Dictionary](https://addons.mozilla.org/firefox/addon/lao-spellchecking-dictionar/)
* [Latin Dictionary](https://addons.mozilla.org/firefox/addon/latin-dictionary/)
* [saskarnes latviskojums](https://addons.mozilla.org/firefox/addon/saskarnes-latviskoju/) (Latvian language pack) and  
  [Latviešu valodas pareizrakstibas parbaudes vardnica](https://addons.mozilla.org/firefox/addon/latvie%C5%A1u-valodas-pareizrakst%C4%ABb/) (Latvian spell checker)
* [Lingala spell checker (Hunspell ln-CD) – Add-ons for Firefox](https://addons.mozilla.org/firefox/addon/lingala-spell-checker/)
* [Lietuviu Language Pack](https://addons.mozilla.org/firefox/addon/lietuvi%C5%B3-language-pack/) and  
  [Lithuanian spelling check dictionary](https://addons.mozilla.org/firefox/addon/lithuanian-spellcheck/)
* [Maithili Language Pack](https://addons.mozilla.org/firefox/addon/maithili-language-pack/)
* [Malagasy Spell Checker](https://addons.mozilla.org/firefox/addon/malagasy-spell-checker/)
* [Te Papakupu Maori](https://addons.mozilla.org/firefox/addon/te-papakupu-m%C4%81ori/) (
Maori/Aotearoa)
* [Marathi Language Pack](https://addons.mozilla.org/firefox/addon/marathi-language-pack/) and  
  [Marathi Dictionary](https://addons.mozilla.org/firefox/addon/marathi-dictionary/)
* [Ndebele (South) spell checker](https://addons.mozilla.org/firefox/addon/ndebele-south-spell-checker/) (isiNdebele)
* [Nepali Dictionary](https://addons.mozilla.org/firefox/addon/nepali-dictionary/)
* [Northern Sotho spell checker](https://addons.mozilla.org/firefox/addon/northern-sotho-spell-checker/) (Sepedi)
* [Correcteur Occitan-Languedocien](https://addons.mozilla.org/firefox/addon/correcteur-occitan-languedocie/)
* [Oriya (OR) Language Pack](https://addons.mozilla.org/firefox/addon/oriya-or-language-pack/),  
  [Oriya Spell Checker](https://addons.mozilla.org/firefox/addon/oriya-spell-checker/) and  
  [Odia Spelling Checker](https://addons.mozilla.org/firefox/addon/odia-spelling-checker/) (or-IN)
* [Lilak, Persian Spell Checker Dictionary](https://addons.mozilla.org/firefox/addon/lilak-persian-dictionary/) (another spell checker and a language pack are already displayed under F for _Farsi_)
* [Punjabi (Gurmukhi) Language Pack](https://addons.mozilla.org/firefox/addon/punjabi-gurmukhi-language-p/) and  
  [Punjabi Spell Checker](https://addons.mozilla.org/firefox/addon/punjabi-spell-checker/)
* [Rumantsch Language Pack](https://addons.mozilla.org/firefox/addon/rumantsch-language-pack/) (Romansh, not to be confused up with Romanian)
* [Sanskrit Spell Checker](https://addons.mozilla.org/firefox/addon/sanskrit-spell-checker/)
* [српски (sr) Language Pack](https://addons.mozilla.org/firefox/addon/%D1%81%D1%80%D0%BF%D1%81%D0%BA%D0%B8-sr-language-pack/) (Serbian),
  [Srpski](https://addons.mozilla.org/firefox/addon/srpski/) (spell checker by dekss) and  
  [Српски речник - Serbian Dictionary](https://addons.mozilla.org/firefox/addon/%D1%81%D1%80%D0%BF%D1%81%D0%BA%D0%B8-%D1%80%D0%B5%D1%87%D0%BD%D0%B8%D0%BA-serbian-dictiona/) (by Љуб. Радовановић)
* [Sinhala (LK) Language Pack](https://addons.mozilla.org/firefox/addon/sinhala-lk-language-pack/)
* [Swati spell checker](https://addons.mozilla.org/firefox/addon/swati-spell-checker/) (Siswati/Swazi)
* [Soŋay Language Pack](https://addons.mozilla.org/firefox/addon/so%C5%8Bay-language-pack/) and  
  [Soŋay senni hantum tenjandikaw](https://addons.mozilla.org/firefox/addon/so%C5%8Bay-senni-hantum-tenjandikaw/) (Songai spell checker)
* [Southern Sotho spell checker](https://addons.mozilla.org/firefox/addon/southern-sotho-spell-checker/) (Sesotho)
* Spanish (Argentina): [Español (AR) Language Pack](https://addons.mozilla.org/firefox/addon/espa%C3%B1ol-ar-language-pack/) and  
[Diccionario español Argentina](https://addons.mozilla.org/firefox/addon/diccionario-espa%C3%B1ol-argentina/)
* Spanish (Chile): [Pack de idioma Español (Chile)](https://addons.mozilla.org/firefox/addon/spanish-chile-language-pack/)
* Spanish (Mexico): [Español (México) Language Pack](https://addons.mozilla.org/firefox/addon/spanish-mexico-language-pack/),  
  [Diccionario español México](https://addons.mozilla.org/firefox/addon/diccionario-espa%C3%B1ol-m%C3%A9xico/) (by eduardo_moreno) and  
  [Diccionario de Español/México](https://addons.mozilla.org/firefox/addon/spanish-mexico-dictionary/) (by Mozilla México, E. Rios, odinmojica)
* Spanish (Spain): [Español (España) Language Pack](https://addons.mozilla.org/firefox/addon/espa%C3%B1ol-espa%C3%B1a-language-pac/) and  
  [Diccionario de Español/España](https://addons.mozilla.org/firefox/addon/spanish-spain-dictionary/)
* Spanish (Venezuela): [Diccionario en español para Venezuela](https://addons.mozilla.org/firefox/addon/diccionario-espa%C3%B1ol-venezuela/)
* [Kiswahili Spell Checker](https://addons.mozilla.org/firefox/addon/kiswahili-spell-checker/) (sw-TZ)
* [Tamil Language Pack](https://addons.mozilla.org/firefox/addon/tamil-language-pack-1/) and  
  [Tamil Spell Checker for Firefox](https://addons.mozilla.org/firefox/addon/tamil-spell-checker-for-firefo/)
* Tamil (India): [ThamiZha! Solthiruthi(Tamil spellchecker)](https://addons.mozilla.org/firefox/addon/thamizha-solthiruthi/)
* [Tsonga spell checker](https://addons.mozilla.org/firefox/addon/tsonga-spell-checker/) (Xitsonga)
* [Tswana spell checker](https://addons.mozilla.org/firefox/addon/tswana-spell-checker/) (Setswana)
* [Turkmen spell checker dictionary](https://addons.mozilla.org/firefox/addon/turkmen-spell-checker/)
* [Uzbek Language Pack](https://addons.mozilla.org/firefox/addon/uzbek-language-pack/) and  
  [Uzbek Spell Checker (Uzbekistan)](https://addons.mozilla.org/firefox/addon/uzbek-spell-checker/)
* [Venda spell checker](https://addons.mozilla.org/firefox/addon/venda-spell-checker/)
* [Venetan](https://addons.mozilla.org/firefox/addon/dithionario-de-lengua-v%C3%A8neta/) (spell checker)
* [Cymraeg Language Pack](https://addons.mozilla.org/firefox/addon/cymraeg-language-pack/) (Welsh) and  
  [Geiriadur Cymraeg](https://addons.mozilla.org/firefox/addon/geiriadur-cymraeg/) (spell checker)
* [Xhosa spell checker](https://addons.mozilla.org/firefox/addon/xhosa-spell-checker/) (isiXhosa)
* [Yiddish spell checker (YIVO)](https://addons.mozilla.org/firefox/addon/yiddish-spell-checker-yivo/)
* [Zulu spell checker](https://addons.mozilla.org/firefox/addon/zulu-spell-checker/)

Also several language packs marked as incompatible are not displayed, but they should be made available at least after setting some compatibility switch (well, this is generally missing at the moment):
* [Akan Language Pack for Firefox](https://addons.mozilla.org/firefox/addon/akan-language-pack-for-firefox/)
* [Aragonese Language Pack](https://addons.mozilla.org/firefox/addon/aragonese-language-pack/)
* [Firefox en valenciano](https://addons.mozilla.org/firefox/addon/valencian-language-pack-for-fi/) (Valencian Catalan language pack)
* [Fijian Language Pack – Add-ons for Firefox](https://addons.mozilla.org/firefox/addon/fijian-language-pack/)
* [Furlan Language Pack](https://addons.mozilla.org/firefox/addon/furlan-language-pack/)
* [Kashubian (CSB) Language Pack](https://addons.mozilla.org/firefox/addon/kashubian-csb-language-pack/)
* [Kurdish (Ku) Language Pack](https://addons.mozilla.org/firefox/addon/kurdish-ku-language-pack/)
* [Ligure (Zeneize) Language Pack](https://addons.mozilla.org/firefox/addon/ligure-zen%C3%A9ize-language-pac/)
* [Luganda (UG) Language Pack](https://addons.mozilla.org/firefox/addon/luganda-ug-language-pack/)
* [Malagasy (MG) Language Pack](https://addons.mozilla.org/firefox/addon/malagasy-mg-language-pack/)
* [Malayalam (IN) Language Pack](https://addons.mozilla.org/firefox/addon/malayalam-in-language-pack/)
* [Ndebele Language Pack](https://addons.mozilla.org/firefox/addon/ndebele-language-pack/)
* [Northern Sotho (Sesotho sa Leboa/Sepedi) Language Pack](https://addons.mozilla.org/firefox/addon/northern-sotho-sesotho-sa-lebo/)
* [Swazi Language Pack](https://addons.mozilla.org/firefox/addon/swazi-language-pack/)
* [Southern Sotho Language Pack](https://addons.mozilla.org/firefox/addon/southern-sotho-language-pack/)
* [Kiswahili (SW) Language Pack](https://addons.mozilla.org/firefox/addon/kiswahili-sw-language-pack/)
* [Kiswahili (SW-TZ) Language Pack](https://addons.mozilla.org/firefox/addon/kiswahili-sw-tz-language-pack/)
* Tamil (Sri Lanka): [Tamil (LK) Language Pack (Outdated)](https://addons.mozilla.org/firefox/addon/tamil-lk-language-pack-old/) and  
  [Tamil (ta-LK) Language Pack](https://addons.mozilla.org/firefox/addon/tamil-ta-lk-language-pack/)
* [Tsonga Language Pack](https://addons.mozilla.org/firefox/addon/tsonga-language-pack/)
* [Tswana Language Pack](https://addons.mozilla.org/firefox/addon/tswana-language-pack/)
* [Venda Language Pack](https://addons.mozilla.org/firefox/addon/venda-language-pack/)
* [Walloon Language Pack](https://addons.mozilla.org/firefox/addon/walloon-language-pack/)
* [Wolof (Senegal) Language Pack](https://addons.mozilla.org/firefox/addon/wolof-senegal-language-pack/)
* [Xhosa Language Pack](https://addons.mozilla.org/firefox/addon/xhosa-language-pack/)
* [Zulu Language Pack](https://addons.mozilla.org/firefox/addon/zulu-language-pack/)

BTW more glitches:
* Why is Albanian sorted as _Shqip_ under S, but Finnish as _Suomi_ under F? (The latter is the mistake I assume.) The same for _Magyar_ (Hungarian) under H. Why is Монгол (Mongol) before Melayu?
* Why sometimes there is a translation (I am German as you guessed), but most of the entries do not have one? This is different in the legacy list.

**Comment by muffinresearch** [comment]
@tofumatt This seems to be an issue that's coming up quite frequently. Is your patch available somewhere so it can be finished off?

**Comment by tofumatt** [comment]
Yeah it’s nearly ready and I can totally push a finished version today.

- Matt (Sent from mobile)

**Comment by muffinresearch** [comment]
> Yeah it’s nearly ready and I can totally push a finished version today.

If this the branch, then we can look to get it ready to save interrupting your time off any more than necessary? https://github.com/mozilla/addons-frontend/tree/language-tools-not-shown-addons/528

**Comment by tofumatt** [comment]
Yup, that's the one. Just pushed a few tweaks to it; there are a few failing tests but easy to fix up. It works on the page though, should just be a matter of tweaking tests. Thanks for taking care of that, sorry I didn't get to it in time! 😄

**Comment by AlexandraMoga** [comment]
Verified on AMO-dev FF56 and FF57 (Win 10)
Results: the new alphabetical sorting based on English vocabulary is working as expected and missing dictionaries have been added to the list:
![image](https://user-images.githubusercontent.com/31961530/32836584-f8bf304c-ca12-11e7-8e48-08599c731051.png)
@muffinresearch : on the old pages, there are 150 languages listed, while on the new front-end there are 125. Differences occur also for Android listings.
Is there a reason for having this discrepancy?

**Comment by muffinresearch** [comment]
@AlexandraMoga I'll check that out.

**Comment by RickieES** [comment]
I've just tested https://addons.mozilla.org/en-US/firefox/language-tools/ with Firefox 56, Firefox 58 and 59, and still can't see the Spanish dictionaries (German appears as Deutsch, but only as language pack). Other Spanish users with Firefox 57 reports the same problem.

**Comment by muffinresearch** [comment]
@RickieES that's because it's not released yet. It's going out today.

**Comment by muffinresearch** [comment]
@AlexandraMoga there are some language packs for locales we don't have represented in the languages list at all, I'm going to file a separate issue to cover that so it can be dealt with in due course.

**Comment by AlexandraMoga** [comment]
Thanks, @muffinresearch 
I'm marking this as verified fixed based on the above comment.
